enhancements/update/cvo-techpreview-manifests.md

```diff
@@ -0,0 +1,154 @@
+---
+title: CVO TechPreview Manifests
+authors:
+  - "@deads2k"
+reviewers:
+  - "@wking"
+approvers:
+  - "@sdodson"
+  - "@wking"
+api-approvers: # in case of new or modified APIs or API extensions (CRDs, aggregated apiservers, webhooks, finalizers)
+  - @sttts
+creation-date: 2021-11-15
+last-updated: 2021-11-15
+tracking-link: # link to the tracking ticket (for example: Jira Feature or Epic ticket) that corresponds to this enhancement
+  - slack thread somewhere
+see-also:
+replaces:
+superseded-by:
+---
+
+# CVO TechPreview Manifests
+
+## Summary
+
+[TechPreviewNoUpgrade](https://github.com/openshift/api/blob/be1be0e89115702f8b508d351c4f5c9a16e5ae95/config/v1/types_feature.go#L32-L34)
+is the canonical way to enable tech preview features.
+It is the setting that all operators use to enable Tech Preview features.
+The CVO will honor this setting on a per-manifest basis, so that tech preview manifests are never present on
+non-tech preview clusters.  
+
+## Motivation
+
+When introducing tech preview operators, the current state of the art (before this enhancement) is to have the 
+operator build an inert mode that detects if tech preview is not enable and then does not take action.
+This approach results in having additional resources, including the clusteroperator/tech-preview itself, 
+installed on every cluster, including non-tech-preview upgrades. 
+This also means that currently, if the tech preview does not progress to stable, the component has to keep "dead" manifests
+around and marked for deletion by the CVO on upgraded clusters.
+By having the CVO honor tech preview per manifest, we can avoid fanning out inert logic to every tech preview operator,
+we can avoid exposing tech preview operators to customers via clusteroperators, and we can avoid creating unnecessary
+resources like namespaces, roles, rolebindings, serviceacounts, and deployments.
+
+### Goals
+
+1. CVO honors an "only create or update this manifest in clusters with tech preview enabled".
```

Comment:
I think it should be `only create and  update instead of `only create or update`. Better if we can write` Create, reconcile and update`